test/main.js

```diff
@@ -48,6 +48,9 @@ function init(callback) {
 		this.seed = this.startingSeed = [0x09d56, 0x08642, 0x13656, 0x03653];
 	};
 
+	// Disable writing to modlog
+	require('./../command-parser.js').CommandContext.prototype.logModCommand = noop;
```

Comment:
And yet, it is needed for some reason, without this I get an error about a bad file descriptor. If I remove mock-fs, the message is output to STDOUT (?).